core/genesis.go

```diff
@@ -184,6 +184,11 @@ func SetupGenesisBlock(db ethdb.Database, genesis *Genesis) (*params.ChainConfig
 	if genesis.Config == nil {
 		return nil, errGenesisNoConfig
 	}
+	// Make sure genesis gas limit is consistent
+	gasLimitConfig := genesis.Config.FeeConfig.GasLimit.Uint64()
```

Comment:
We could choose to only do this on a go-forward basis. (i.e. if genesis doesn't exist)

Reply:
This is obviously not a great solution because old validators can continue fine but new validators won't be able to join.